requirements.txt

```diff
@@ -1,8 +1,8 @@
 numpy>=1.0
 scipy>=1.0
 matplotlib>=3.0
-netCDF4>=1.4,<1.6; python_version == "3.7"
-netCDF4>=1.6.2; python_version != "3.7"
+netCDF4>=1.4,<1.6; python_version < "3.11"
```

Comment:
The old script didn't make sense to me, this one is more aligned with the requirements definition made in setup.py

Reply:
Yeah... so it seems python 3.7 requires `netCDF4>=1.4,<1.6`. Since we are dropping 3.7 we can just remove the requirement